vrp/modules/paycheck.lua

```diff
@@ -1,133 +1,145 @@
 local Proxy = module("vrp", "lib/Proxy")
```

Comment:
You don't need this line. Please remove.

Reply:
Was in default so just left it but will do